### ISSUE TITLE
regression test failure with github-markdown-toc 1.2.0

### ISSUE THREAD

**Issue by chenrui333** [ekalinin/github-markdown-toc.go#39]
# Problem

Run into some regression test failure when trying to build the new bottle for macOS Ventura, the error is the heading processing is not working as expected.

# How to reproduce

- `brew install github-markdown-toc`
-`echo "# Header" > README.md`
- `gh-md-toc --debug README.md`

```
$ gh-md-toc README.md

Table of Contents
=================


Created by [gh-md-toc](https://github.com/ekalinin/github-markdown-toc.go)
```

Debug mode

```
|$ gh-md-toc --debug README.md

Table of Contents
=================

2022/11/07 21:16:37 Convert2HTML: start.
2022/11/07 21:16:37 IsRemoteFile: false
2022/11/07 21:16:37 Convert2HTML: local file: README.md
2022/11/07 21:16:37 Convert2HTML: converted to html, size: 237
2022/11/07 21:16:37 Convert2HTML: write html file: README.md.debug.html
2022/11/07 21:16:37 Convert2HTML: done.
2022/11/07 21:16:37 GrabToc: start, html size: 237
2022/11/07 21:16:37 GrabToc: matching ...
2022/11/07 21:16:37 GrabToc: processing groups ...
2022/11/07 21:16:37 Including starting frome level 0
2022/11/07 21:16:37 GrabToc: done.

Created by [gh-md-toc](https://github.com/ekalinin/github-markdown-toc.go)
```

**Comment by chenrui333** [comment]
Thanks @cgrindel!

**Comment by ekalinin** [comment]
Could you, please, try 1.2.1 version I just released:

https://github.com/ekalinin/github-markdown-toc.go/releases/tag/v1.2.1

**Comment by ekalinin** [comment]
Should be fixed in 1.2.1
